example/src/CM/EditView/Information.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Box } from "@strapi/design-system/Box";
 import { Stack } from "@strapi/design-system/Stack";
-import { Text } from "@strapi/design-system/Text";
+import { Typography } from "@strapi/design-system/Typography";
```

Comment:
WDYT about deleting the example app completely we don't need it anymore.

Reply:
Can we do that in another PR ?

Reply:
Done in #423 can you update your PR?